DayrSuryan/DSEthiop1.xml

```diff
@@ -664,7 +664,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <bindingDesc>
                      <binding xml:id="binding">
                         <decoNote xml:id="b1">
-                           Two wooden boards covered with reddish brown blind-tooled leather.
+                           Bound in Ethiopia. Two wooden boards covered with reddish brown blind-tooled leather.
```

Comment:
is there a conceptual difference between "Bound in Ethiopia" here and "Ethiopian" elsewhere?

Reply:
It should be "Ethiopian binding".